registry/quilt_server/models.py

```diff
@@ -244,3 +245,8 @@ class Comment(db.Model):
     contents = db.Column(db.String(MAX_COMMENT_LENGTH), nullable=False)
 
     package = db.relationship('Package', back_populates='comments')
+
+class Role(db.Model):
+    id = db.Column(postgresql.UUID, primary_key=True)
+    name = db.Column(db.String(200), unique=True, nullable=False)
+    arn = db.Column(db.String(200), nullable=False)
```

Comment:
I think arn should be unique. Otherwise, users can create aliases for the same role. Unless there's a reason to support that?

Reply:
I don't see any reason to make a rule that they can't alias -- we could warn when creating an aliasing role but I don't think it's a correctness issue either way and it's simpler to allow them to alias